src/modules/app/reducers/index.js

```diff
@@ -5,8 +5,12 @@ import {
   CITIES_ENDPOINT_NAME,
   DISCLAIMER_ENDPOINT_NAME,
   EVENTS_ENDPOINT_NAME,
+  LOCALNEWS_ENDPOINT_NAME,
+  LOCALNEWS_ELEMENT_ENDPOINT_NAME,
+  TUNEWS_ELEMENT_ENDPOINT_NAME,
```

Comment:
@amusameh are we missing tunews list endpoint here?

Reply:
No we are not missing it. tunews list has its own reducer

Reply:
tunews list needed a separate reducer to handle the pagination. the general reducer works for the rest